scripts/g.extension/g.extension.py

```diff
@@ -1347,11 +1347,11 @@ def download_source_code(source, url, name, outdev,
         # we expect that the module.zip file is not by chance in the archive
         zip_name = os.path.join(tmpdir, 'extension.zip')
         try:
-            response = urlopen(url)
+            response = urlopen(Request(url, headers={'User-Agent': 'Mozilla/5.0'}))
         except URLError:
             grass.fatal(_("Extension <%s> not found") % name)
         with open(zip_name, 'wb') as out_file:
-            shutil.copyfileobj(response, out_file)
+            out_file.write(response.read())
```

Comment:
Just curious, was there are problem with the previous code or is this just for more readability?